presto-cli/pom.xml

```diff
@@ -126,6 +126,19 @@
             <artifactId>mockwebserver</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <!-- for testing -->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <scope>test</scope>
+            <exclusions>
```

Comment:
I had to add this exclusion because jline-terminal-jna pulls in the same artifact but with a different version.
Does this look ok? Or should this be handled differently?